docs/modules/release-notes/nav.adoc

```diff
@@ -1,4 +1,7 @@
 * Release notes
+
```

Comment:
Remove line `2`, so there is no new line between `* Release notes` and `** 2023`.